### DISCUSSION TITLE
"How Do It Know" build and fail

### DISCUSSION THREAD

**Discussion by medennis34** [hneemann/Digital#1160] [Q&A]
Greetings, I have enjoyed using Digital as I study for my A+ Cert. Thx! After reading "How Do It Know", I found Digital and started building. My two iterations, one with gates under the "components" menu and a second with gates from FETs in the examples file, both  die an oscillation death.

![OSCILLATE](https://github.com/hneemann/Digital/assets/132152600/32cbd837-ca79-4f76-9f58-57ab19a03422)

The accompanying screen shot shows the FETs themselves oscillating from the second build. That fix might be above my pay grade (lol). 
I got a similar message in my first build using the components gates. Strangely, when I test an embedded circuit isolatively, all is fine.

Here are my library files, any advice on how to be successful with this project would be great. Thx in advance.
[Michaels_PC_zip.zip](https://github.com/hneemann/Digital/files/11536557/Michaels_PC_zip.zip)
[HDIK_zip.zip](https://github.com/hneemann/Digital/files/11536575/HDIK_zip.zip)

**Comment by hneemann** [comment]
I have not looked at the circuits, but oscillation always has the same cause: 
The circuit is essentially a feedback inverter:

![z](https://github.com/hneemann/Digital/assets/7190563/d8173cf4-dd6b-4d81-854e-9d5442d5c850)

And such circuits never reach a stable state and oscillate forever.
